docs/docs/guides/how-to/expired-session-cleanup/index.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
I think we should have a more general topic of "how to run scheduled tasks" with a general explanation of the problem and solution and then the session cleanup as the main example. In future we can then add more examples like cleaning up abandoned orders.

Reply:
good idea! should I put that into the "Developer Guide" section under "Advanced Topics" just like the "Migrating from v1" topic? 
<img width="255" alt="image" src="https://github.com/user-attachments/assets/43ebf119-38ea-477d-9d23-3a927ee40b3a">

Reply:
Yes, good idea 👍